docs/file_format.rst

```diff
@@ -38,6 +38,13 @@ The optional ``extracellular`` key specifies the default string for
 the extracellular compartment on compounds. If this option is not
 specified it will be assumed that the extracellular compartment is called ``e``.
 
+-Default Compartment
+--------------------------
+
+The optional ``default`` key specifies the default string for
+the default compartment on compounds. If this option is not
```

Comment:
Also, I think we should provide an example. Something like: "For example, the reaction `|x[e]| + |atp| => |x| + |adp| + |pi|` does not specify a compartment on four of the compounds so these would automatically be presumed to appear in the default compartment (or `c` if no default compartment is specified)."